README.md

```diff
@@ -160,9 +162,9 @@ The solution uses Search as primary tool for customer interaction. The customers
 
 1. Supports multiple search engines: Elastic Search, Elastic Cloud, Azure Search and Lucence;
 
-1. Functionality is exposed through REST API or .NET SDK that makes the inherent complexity of information retrieval;
+1. Functionality, which implements the inherent complexity of information retrieval is exposed through REST API or .NET SDK;
 
-1. The support documents can be easily extended by third party team.
+1. The support document types (Product, Category, Member) can be easily extended by third party team.
```

Comment:
Did you mean "Supported document types"?

Reply:
yes!